samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/ble_mesh.c

```diff
@@ -51,7 +51,7 @@ static const struct bt_mesh_prov prov = {
 	.reset = prov_reset,
 };
 
-void bt_ready(int err)
+void bt_ready(void)
 {
 	struct bt_le_oob oob;
 
```

Comment:
CI will probably fail since you're still using err inside this function, i.e. you'll need to add a local err variable

Reply:
yes !!

Reply:
have just push new request